### PR TITLE
:art: Image input block redesign block redesign #639

### LINKED ISSUE #639: :lipstick:  Image Input Block Redesign: block redesign
## User stories

As a user, I want the "Image Input Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/image-input-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/image-input-block/`.

### PR DESCRIPTION
**Pull request by thadeusdev**
# Description

- changed image icon with the lighter version of it in svg.
- Adjusted font size and font weight for the block title.
- changed the ellipsis-v icon with the light version of its svg.
- styled the input field i.e background color, border-radius, font-weight, font-size and set outline to none.
- removed the hr tag that was adding an extra horizontal line.
- refactored jsPlumb

Fixes #639

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screenshot from 2023-11-02 21-17-56

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings

### CODE REVIEW

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/image-input-block.jsplumb.ts [hunk 1]
```diff
@@ -19,10 +19,10 @@ export function _ImageInputBlockDecoratePlumb(block: ImageInputBlock, comp: Comp
   jsPlumb.addEndpoint(comp.location.nativeElement, {
     // Whether the anchor is target (Other Block -> This Block)
     target: true,
-    cssClass:"block_endpoint",
-    endpoint: "Dot",
-    anchor: "Left",
-    maxConnections: -1
+    cssClass: 'block_endpoint',
+    endpoint: 'Dot',
+    anchor: [0, 0.2, 0, 0],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/image-input-block.jsplumb.ts:24]: Have this as 'Left'

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync the latest changes and then resolve conflicts. Also, kindly edit your PR scope on your title as this is a design feature and not ci related.

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/image-input-block.jsplumb.ts (hunk 1 above)
Comment (thadeusdev) [reply libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/image-input-block.jsplumb.ts]: Noted. I have done as advised. Thank you for the heads up 🙏

**Comment by LemmyMwaura** [comment]
@thadeusdev resolve conflicts so we can merge

**Comment by thadeusdev** [comment]
conflicts resolved

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**